src/cookies/getCookie.ts

```diff
@@ -20,7 +20,7 @@ export const getCookie = ({
 
 	const cookieVal = getCookieValues(name);
 
-	if (cookieVal.length > 0) {
+	if (cookieVal[0]) {
```

Comment:
i wonder if we could shift the array access up to `const cookieVal = getCookieValues(name)[0];` for consistency with https://github.com/guardian/libs/pull/390/files#diff-fed7a7bee596fcd23f90f5770a22a3d2828ce9d22c8a0f46518c6c6715798498R48 for example?